src/dialogs/notifications/notification-drawer.js

```diff
@@ -111,10 +123,24 @@ export class HuiNotificationDrawer extends EventsMixin(
     this.open = false;
   }
 
+  _dismissAll(ev) {
+    ev.stopPropagation();
```

Comment:
Is this needed?

Reply:
I followed the practice from somewhere else in the file. I don't think it's needed. Happy to remove it.

Reply:
```suggestion
```